scripts/run-clang-tidy.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env python
```

Comment:
I guess this file is copied from the LLVM project (or somewhere else) - does it need to be in the doctest repository? Can't it be used from wherever clang-tidy is located? what about the license?

Reply:
Not shure, but not every distribution install it (windows, mac ports, ...)
Debian does.